rest_test/test-results

```diff
@@ -0,0 +1,181 @@
+[32mpass[39m: GET (200) /node_info duration: 79ms
```

Comment:
Is there a reason to keep this file? If not I'd remove it to keep things tidy.

Reply:
No reason. I will remove.